.github/ISSUE_TEMPLATE/course-template.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Course Template
+about: Describe this issue template's purpose here.
+title: Course - TUTORIAL TITLE
+labels: Awaiting Triage, [Content Type] Course
+assignees: ''
+
+---
+
+## Needs Assessment Statement
+<!-- A needs assessment statement is a summary of the gaps in knowledge and skills that the course will address. It helps identify the actual learning needs of the students, align course objectives with the desired outcomes, and ensure the right type of content is used to optimize learning effectiveness and engagement. -->
+
+## Target Audience
+<!- Describe the specific group of learners for whom the course is designed and intended. These learners usually share common characteristics, skill level, or professional goals, which enable the course creator to tailor the content and learning objectives to suit their needs. -->
+
+## Course Objective
+<!-- The course objective is a statement that describes the expected learning outcomes of the course. It is a concise statement that describes the knowledge and skills that the learners will acquire after completing the course. -->
+
+## Outline <a href="#outline" id="outline">#</a>
+<!-- The outline is a list of the topics that will be covered in the course. It is a high-level overview of the course content. Feel free to include links to exsiting content that will be used in the course. -->
+
+- [ ] https://learn.wordpress.org/tutorial/using-the-create-block-tool/
+- [ ] https://learn.wordpress.org/tutorial/converting-a-shortcode-into-a-block/
+- [ ] https://learn.wordpress.org/tutorial/styling-your-wordpress-block/
+- [ ] https://learn.wordpress.org/tutorial/using-block-attributes-to-enable-user-editing/
+
+# Related Resources
+Links to related content on Learn, HelpHub, DevHub, GitHub Gutenberg Issues, DevNotes, etc.
+- [ ]
+
+# Guidelines
+Review the [team guidelines](https://make.wordpress.org/training/handbook/guidelines/)
+
+# Course Development Checklist
+
+- [ ] Needs Assessment Statement, Target Audience, Course Objective, and Outline [created](https://make.wordpress.org/training/handbook/courses/content-outline-and-draft/) 
+- [ ] Vetted by instructional designers for content idea
+- [ ] Provide feedback of the idea
```

Comment:
Is this step needed? If so, who is giving feedback? This seems similar to the point above.

Reply:
I think it depends on if someone is suggesting this as an idea for a course, vs someone is creating the ticket to create the course. So someone might create the issue, with the Needs Assessment Statement, Target Audience, Course Objective, and Outline, but they might want IDs/SMEs to vett and provide feedback on the idea, ie does the Needs Assessment Statement, Target Audience, Course Objective, and Outline make sense, does the course make sense etc.

Reply:
I'm still confused... If "Vetted by instructional designers for content idea" is done by IDs/SMEs, then who is "Provide feedback of the idea" done by? That seems redundant to me here here.

Reply:
@bsanevans updated the initial "review" steps to be clearer as to who does what.